modular_skyrat/modules/modular_items/lewd_items/code/arousal_system.dm

```diff
@@ -345,17 +345,17 @@
 
 /datum/mood_event/orgasm
 	description = span_purple("Woah... This pleasant tiredness... I love it.\n")
-	mood_change = 8 //yes, +8. Well fed buff gives same amount. This is Fair (tm).
+	mood_change = 0 //8 is not fair(tm)
```

Comment:
```suggestion
```
Parent value is 0. Apply to all.

Obligatory based committing from master comment

Reply:
good ideay